cc3d/player5/Configuration/ConfigurationDialog.py

```diff
@@ -697,9 +735,14 @@ def updatePreferences(self):
         Configuration.setSetting("WriteMovieMCS", self.writeMCSCheckbox.isChecked())
         Configuration.setSetting("FfmpegLocation", self.ffmpegLocationLineEdit.text())
 
+        # Miscellany/Setup
         if str(self.outputLocationLineEdit.text()).rstrip() == '':
             Configuration.setSetting("OutputLocation", os.path.join(os.path.expanduser('~'), 'CC3DWorkspace'))
 
+        Configuration.setSetting("ThemeIndex", self.themeComboBox.currentIndex())
```

Comment:
Can we try using just one Setting - ThemeName? It looks like we should be able to get away with this. Adding ThemeIndex may seem convenient at first glance but it will be harder to maintain two settings that essentially describe one entity.  the theme name.

Reply:
Yes, I will try

Reply:
@maciekswat It was no problem. I tested with ``DELETE FROM settings where name='ThemeIndex';`` on my SQLite file, and it works.